lib/manager/actions/versions.js

```diff
@@ -160,6 +160,16 @@ export function fetchFeedVersion (feedVersionId: string) {
  */
 export function publishFeedVersion (feedVersion: FeedVersion) {
   return function (dispatch: dispatchFn, getState: getStateFn) {
+    if (
+      feedVersion.feedSource.autoPublish &&
+      feedVersion.sentToExternalPublisher &&
+      !window.confirm('Are you sure you want publish this feed version again?')
+    ) {
+      // For a feed source set to auto-publish,
+      // if the version has already been sent to publishing
+      // and the user does not confirm, do nothing.
+      return
+    }
```

Comment:
I'm going to remove this block because the Publish button is already disabled when the version is published (you cannot publish twice).